packages/localization/src/languages/pl.json

```diff
@@ -4,9 +4,14 @@
             "devices": "Urządzenia",
             "certificates": "Certyfikaty",
             "demands": "Oferty",
-            "organizations": "Organizacje"
+            "organizations": "Organizacje",
```

Comment:
It's okay but no need to add corresponding keys to other languages than `en.json` for now. If translation key is not present in other language than EN, it will fallback to EN. :)